src/main/java/edu/greenblitz/tobyDetermined/subsystems/telescopicArm/Elbow/Elbow.java

```diff
@@ -127,9 +128,9 @@ public double getLegalGoalAngle(double angleInRads){
         if (getHypotheticalState(angleInRads) == ElbowState.FORWARD_OUT_OF_BOUNDS || getHypotheticalState(angleInRads) == ElbowState.BACKWARD_OUT_OF_BOUNDS){
             Console.log("OUT OF BOUNDS", "arm Elbow is trying to move OUT OF BOUNDS" );
```

Comment:
Use the logger

Reply:
there is a diffenece between console.log to the logger, is that the console.log puts it in the drivers dashboard

Reply:
we also log when the arm is moving out of bounds